### PR TITLE
DSM-2025 - Des Moines 2025 Initial Setup

### PR DESCRIPTION
**Pull request by jangolano**
DSM-2025 - Des Moines 2025 Initial Setup
Initial Creation of 2025 Site
Added new organizers, sent an email about the new organizers

### CODE REVIEW

File: static/events/2025-des-moines/organizers/akrem-saed.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/akrem-saed.jpg:1]: Please move this image to the new asset folder`assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/amy-lindberg.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/amy-lindberg.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/bo-schatzberg.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/bo-schatzberg.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/crystal-newcomb.png
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/crystal-newcomb.png:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/willie-sandry.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/willie-sandry.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/tim-gifford.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/tim-gifford.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/ted-wallace.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/ted-wallace.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/paul-beers.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/paul-beers.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/josh-angolano.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/josh-angolano.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/john-sonneville.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/john-sonneville.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/jodi-jones.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/jodi-jones.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/ethan-spoelstra.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/ethan-spoelstra.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: static/events/2025-des-moines/organizers/joanna-springsteen.jpg
Comment (toshywoshy) [inline static/events/2025-des-moines/organizers/joanna-springsteen.jpg:1]: Please move this image to the new asset folder `assets/events/2025-des-moines/organizers`

File: data/events/2025/des-moines/main.yml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    linkedin: "https://www.linkedin.com/in/sonnevillej/"
+    twitter: "agile_johnville"
+  - name: "JoAnna Springsteen"
+    image: 'joanna-springsteen.jpg'
+    linkedin: "https://www.linkedin.com/in/joannaspringsteen/"
+  - name: "Ted Wallace"
+    image: 'ted-wallace.jpg'
+    linkedin: "https://www.linkedin.com/in/ted-wallace-9299757/"
+organizer_email: "des-moines@devopsdays.org" # Put your organizer email address here
+
```
Comment (toshywoshy) [inline data/events/2025/des-moines/main.yml:97]: Please send an email to info@

**Review by toshywoshy — CHANGES_REQUESTED** [review]

File: data/events/2025/des-moines/main.yml (hunk 1 above)
Comment (jangolano) [reply data/events/2025/des-moines/main.yml:97]: info email has been sent

**Review by toshywoshy — CHANGES_REQUESTED** [review]
Email info@ for updates

**Comment by adrianmoisey** [comment]
> Email info@ for updates

Email received and actioned.
I haven't checked the rest of this PR, but that part is now unblocked

**Review by toshywoshy — APPROVED** [review]

**Pull request merged**